profiles/bicycle.lua

```diff
@@ -93,8 +93,8 @@ surface_speeds = {
 
 -- these need to be global because they are accesed externaly
 properties.traffic_signal_penalty        = 2
-properties.use_turn_restrictions         = false
 properties.u_turn_penalty                = 20
+properties.use_turn_restrictions         = false
```

Comment:
Yeah so that it follows the order in the property_profiles. Only did this because I'm bringing in max_speed with my next PR and I wanted them in order :p